config.ru

```diff
@@ -10,6 +10,8 @@
 # More conversation at https://github.com/edx/cs_comments_service/pull/146
 # -Nov 18th, 2015
 
+require "newrelic_rpm"
```

Comment:
@jibsheet @tobz: This line fixed the traces in X-Ray, but "may" be causing issue with db monitoring???  I need to experiment and/or open a ticket with New Relic again.